cluster-autoscaler/proposals/cp_agnostic_ca.md

```diff
@@ -0,0 +1,158 @@
+# Decoupling ClusterAutoScalerCore (separate library) from CloudProvider implementations
```

Comment:
Please, update the title accordingly to reflect the new approach

Reply:
Done